contributors/guide/code-review.md

```diff
@@ -0,0 +1,275 @@
+    - [Commit Hygiene](#commit-hygiene)
+  - [Be clear on progress and time](#be-clear-on-progress-and-time)
+  - [Checking out a PR](#checking-out-a-pr)
+- [Additional Resources](#additional-resources)
+
+
+## Tips for getting your PR reviewed
+
+
+### Use Tokens to signal the state of your PR
+
+Token phrases like `WIP` or `do not merge` signal to reviewers that the PR is
+potentially in a state that is not complete. This can save significant time for
+reviewers on prioritizing which PR should be reviewed. They can either hold off
+on reviewing altogether or will review the PR with the knowledge that it is not
+complete.
+
+
+### Reduce the scope of your PR
+
+Extremely large PRs take significant amounts of time to both review and evaluate
+for potential upstream/downstream impacts. If it is possible to achieve the
+desired goal over multiple smaller, more concise PR("Pre-factor"), it decreases
+risk and will have a greater chance of being accepted.
+
+
+### Split your PR into logical smaller commits
+
+Multiple smaller commits that contain the logical pieces of work simplify the
+review process for others to review your code. These commits should be able to
+stand on their own and not necessarily depend on other things.
+
+**Example: Markdown**
+
+A PR that applies both formatting changes and content changes. If taken together
+in a single commit, it will appear to your reviewer that the entire document has
+changed and will require more time to compare and separate the actual content
+changes from the style changes.
+
+If content and style changes were made in separate commits, they could be reviewed
+independently for their changes.
+
+**Example: Generated code**
+
+A PR that has both authored code and generated code can be difficult to parse.
+If the authored code and generated code are in separate commits, the generated
+code can largely be ignored by the reviewer, and they can focus their time on
+the authored code.
+
+### Squash or remove unnecessary commits
+
+Often you may rapidly commit small changes or fixes. These are even more common
+when accepting suggestions through the GitHub UI where each change is done as an
+independent small commit. [Squashing] these down into a logical commit will both
+help your reviewer when performing a final pass, and provide for a clean git
+commit history.
+
+
+### Explain your changes (comments)
+
+Comments should be used for **ANY** non-obvious changes. They should explain why
+something was done and the intent behind it; leave no surprises for the next
+person to view or touch your code. If you spent time debugging, testing, or 
+searching for prior art on a means to do something - document it in the comments.
+
+If you leave a `TODO`, it should explain why it needs to be revisited along with
+any relevant information that could help the next person looking to update that
+part of the code. Consider following it up with a help wanted issue if appropriate.
```

Comment:
suggestion: "Consider following insertion of a `TODO` with also creating a new help wanted issue, if appropriate."

Reply:
done :)